### PR TITLE
Added support for <propertypattern>

### PR DESCRIPTION
**Pull request by bambi73**
Added support for custom shortcut properties patterns. 
Feature is described in Advanced Usage.txt point 14.

### CODE REVIEW

File: resources/lib/xmlfunctions.py [hunk 1]
```diff
@@ -717,13 +717,61 @@ def buildElement( self, item, Tree, groupName, visibilityCondition, profileVisib
...
+        propertyPatterns = {}
+        
+        if overrides is not None:
+            propertyPatternElements = overrides.getroot().findall("propertypattern")
+            for propertyPatternElement in propertyPatternElements:
+                propertyName = propertyPatternElement.get("property")
+                propertyGroup = propertyPatternElement.get("group")
+              
+                if propertyName and propertyGroup and propertyPatternElement.text:
+                    if propertyGroup == group:
```
Comment (phil65) [inline resources/lib/xmlfunctions.py]: would be nice you would do "if not ...: continue" instead of super-deep nesting :)

**Comment by phil65** [comment]
looks fine apart from cosmetics (I prefer to use continue instead of deep nesting of conditionals)

**Comment by bambi73** [comment]
Reversed conditions and added continue to remove some nesting

**Comment by phil65** [comment]
thx.

**Pull request merged**